phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/ControllerConfiguration.java

```diff
@@ -143,4 +143,8 @@ public SettingsController settingsController() {
                 settingsConfiguration.settingsManager()
         );
     }
+
+    public String getWineEnginesPath() {
+        return this.wineEnginesPath;
```

Comment:
That's not what I had in mind. I think you should replace `this.wineEnginesPath` with `this.enginesPath` which shall be `application.user.engines`. Then you can use `enginesPath + engineCategory.getName()` to determine the path for the special engine (like Wine).
I know that this might interfere with `application.user.engines.wine` but I think that's a different topic. In my opinion, we should just remove those engine specific properties. If we can't do that, we have to create a generic interface which allows you to query the installation path from JS.